src/rest-api/trainer/trainerMockData.ts

```diff
@@ -0,0 +1,12 @@
+import {EditTrainingSummary} from '../../model/editTrainingSummary';
+
+export const trainingContentMockData: EditTrainingSummary[] = [
+  {
+    id: 1,
+    content: 'This is a test markdown sample using # H1',
```

Comment:
Just use backticks and let's copy some readme.md that we have already, or just figure out the master TOC, I think it would be nice to see just how it behaves.

I would say id 1 the full thing, id 2 just keep it as one line